packages/util-package-extender/lib/js/_utils/_get-remote-file.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * _get-remote-file.js
+ * Get the contents of a remote file
+ * Used to get package contents from CDN
+ */
+'use strict';
+
+const got = require('got');
+
+// Number of times to retry request
+const maxRetry = 2;
+
+// Request configuration options
+const config = {
+	retry: {
+		retries: maxRetry,
+		statusCodes: new Set([404, 408, 413, 429, 500, 502, 503, 504]),
+		errorCodes:
+			new Set([
+				'ETIMEDOUT',
+				'ECONNRESET',
+				'EADDRINUSE',
+				'ECONNREFUSED',
+				'EPIPE',
+				'ENOTFOUND',
+				'ENETUNREACH',
+				'EAI_AGAIN'
+			])
+	},
+	hooks: {
+		beforeRetry: [
+			(options, error, retryCount) => {
+				console.log(`fail: requesting ${options.href}`);
+				console.log(`info: ${generateErrorMessage(error)}`);
+				console.log(`info: retrying (${retryCount}/${maxRetry})`);
+			}
+		]
+	}
+};
+
+/**
+ * Generate an error message based on error type
+ * @private
+ * @param {Object} error the error object from the request
+ * @return {String}
+ */
+function generateErrorMessage(error) {
+	switch (error.name) {
+		case 'RequestError':
+			return `request failed with ${error.code}`;
+		case 'HTTPError':
+		case 'MaxRedirectsError':
+			return `${error.statusCode} (${error.statusMessage})`;
+		default:
+			return `Uh-oh Something went wrong`;
```

Comment:
Add the error.name
```suggestion
			return `Uh-oh Something went wrong. The unhandled error was ${error.name}`;
```

Also, inconsistent casing! :twitch: